### DISCUSSION TITLE
Welcome to PiOS Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/PiOS#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tranduyhung1987** [comment]
![Uploading Screenshot_20250925-142411.jpg…]()
Thưa ông. Điều gì đang xảy ra với các giao dịch mainnet.
